src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -104,7 +105,8 @@ public EventTypeService(final EventTypeRepository eventTypeRepository,
             eventType.setTopic(topicName);
             eventTypeRepository.saveEventType(eventType);
             return Result.ok();
-        } catch (final InvalidEventTypeException | NoSuchPartitionStrategyException e) {
+        } catch (final InvalidEventTypeException | NoSuchPartitionStrategyException |
+                DuplicatedEventTypeNameException e) {
```

Comment:
Could you please change log in line "113/115" to `error` since this is leaving the system in an inconsistent state, i.e. orphan topic in Kafka without corresponding event type in DB.